protocol/blockchain/encoding/cashaddr.md

```diff
@@ -0,0 +1,201 @@
+# CashAddr encoding
+
+CashAddr encoding is an address format used in Bitcoin Cash. This is a Base32 encoding format that prevents confusion with [Base58Check](/protocol/blockchain/encoding/base58check) legacy address encoding also used by Bitcoin-BTC. The goal of the encoding is to make it easier to copy and to share information, by using a QR code for instance.
+
+A Cash Address consists of:
+
+- A **prefix** which is human-readable.
+- A **separator** (`:`).
+- A Base32 **payload** which contains a version byte and the data (or hash).
+- A Base32 **checksum**.
+
+This format reuses the work done for Bech32 (see [BIP173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki)) and is similar in some aspects, but improves on others. See the [original specification](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/cashaddr.md) for more details.
+
+## Prefix
+
+The prefix is a human-readable part of the address which indicates the network on which the addess is valid. It can only contain ASCII characters. The 3 prefixes used in Bitcoin Cash are:
+
+| Network | Prefix        |
+| ------- | ------------- |
+| Mainnet | `bitcoincash` |
+| Testnet | `bchtest`     |
+| Regtest | `bchreg`      |
+
+The prefix is always followed by the separator `:`.
+
+When presented to users, the prefix and the separator may be omitted as it is part of the checksum computation. 
+
+## Base32
+
+CashAddr uses Base32 to encode information. The symbols used in CashAddr Base32 are the lowercase alphanumeric characters excluding `1`, `b`, `i`, and `o`. 
+
```

Comment:
All upper case is also valid (and allows for efficient QR code encoding)

Reply:
Yes, I missed that. In the [CashAddr spec](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/cashaddr.md#uppercaselowercase):

> Lower case is preferred for cashaddr, but uppercase is accepted. A mixture of lower case and uppercase must be rejected.
>
> Allowing for uppercase ensures that the address can be encoded efficiently in QR codes using the alphanumeric mode.

I will add that next time, along with the `simpleledger` thing, and correction of typos.